### PR TITLE
Gen AI lab: styling pass

### PR DESCRIPTION
**Pull request by bencodeorg**
## Warning!!

The [AP CSP Create Performance Task](https://apcentral.collegeboard.org/courses/ap-computer-science-principles/exam) is in progress. The most critical dates are from April 3 - April 30, 2024. Please consider any risk introduced by this PR that could affect our students taking AP CSP. Code.org students taking AP CSP primarily use App Lab for their Create Task, however a small percent use Game Lab. Carefully consider whether your change has any risk of alterering, changing, or breaking anything in these two labs. Even small changes, such as a different button color, are considered significant during this time period. Reach out to the Student Learning team or Curriculum team for more details.

## Description

Friday afternoon timeboxed glow up 💅 for gen AI:
- restyled copy button
- blue border on `textarea` and `input`, standardize sizes
- styling of added retrievals, with red x on hover to remove
- smaller toggle button to get into presentation view
- set max temperature to 1 with input from Dan and Tyrone

| Update | "After" screenshot |
| - | - |
| Updated color/size on inputs | image |
| Updated copy chat button | image |
| Smaller toggle | image |
| Updated retrieval tab | image |

### CODE REVIEW

File: apps/src/aichat/views/AichatView.tsx [hunk 1]
```diff
@@ -25,22 +25,6 @@ import {AichatLevelProperties, ViewMode} from '@cdo/apps/aichat/types';
 import {isProjectTemplateLevel} from '@cdo/apps/lab2/lab2Redux';
 import ProjectTemplateWorkspaceIcon from '@cdo/apps/templates/ProjectTemplateWorkspaceIcon';
 
-const renderChatWorkspaceHeaderRight = (onClear: () => void) => {
```
Comment (bencodeorg) [inline apps/src/aichat/views/AichatView.tsx:31]: Tired of seeing helpers at the top of React components, so moved to bottom :)

File: apps/src/aichat/views/model-customization-workspace.module.scss [hunk 2]
```diff
@@ -13,13 +13,19 @@
...
     margin: 0;
+    border-radius: 4px;
+    border-color: $cyan;
   }
 
   input {
     @extend %full-width;
+    height: 30px;
+    border-radius: 4px;
+    border-color: $lighter_gray;
```
Comment (fisher-alice) [inline apps/src/aichat/views/model-customization-workspace.module.scss]: Posted a question about this to @markabarrett on [Slack](https://codedotorg.slack.com/archives/C06FELVTV0Q/p1713210622623339?thread_ts=1712006936.379659&cid=C06FELVTV0Q).

**Review by sanchitmalhotra126 — APPROVED** [review]
✨ 💅 🧹

**Review by fisher-alice — APPROVED** [review]
Awesome! Left a question about input box border which I believe Mark responded to already. Thanks!

File: apps/src/aichat/views/model-customization-workspace.module.scss (hunk 2 above)
Comment (bencodeorg) [reply apps/src/aichat/views/model-customization-workspace.module.scss]: Didn't fully implement the design system for inputs and textareas, but at least switched the outline to match the preferred color (`$light_black`)

**Pull request merged**